### PR TITLE
Add support for content components

### PR DESCRIPTION
**Pull request by brenetic**
## Make metadata updater to accept a new component

Add all components for each page

Every time we add a new component we need to make sure that the component ids (and item ids) are unique.

## Update metadata update spec to use page fixture with no component

Co-authored-by: Tomas Destefi <tomas.destefi@digital.justice.gov.uk>
Co-authored-by: Natalie Seeto <natalie.seeto@digital.justice.gov.uk>

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -36,6 +36,7 @@ class Application < Rails::Application
 
 Sentry.init do |config|
   config.breadcrumbs_logger = [:active_support_logger]
+  config.logger =  Logger.new(STDOUT)
```
Comment (tomas-stefano) [inline config/application.rb:39]: Should it be `Rails.logger`?

**Review by tomas-stefano — APPROVED** [review]

File: config/application.rb (hunk 1 above)
Comment (brenetic) [reply config/application.rb:39]: at this point Rails.logger doesn't exist

**Review by njseeto — DISMISSED** [review]
🚀

**Review by tomas-stefano — DISMISSED** [review]
🎸 🚢

**Review by tomas-stefano — COMMENTED** [review]
🚢

**Review by tomas-stefano — APPROVED** [review]
🚢

**Pull request merged**